### PR TITLE
Publish 1.2.2

### PR DESCRIPTION
**Pull request by itowlson**

### CODE REVIEW

**Review by bnookala — APPROVED** [review]
oh wow, a callout in the changelog 🤗

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,28 @@
 # Change Log
 
+## 1.2.2
```
Comment (lstocchi) [inline CHANGELOG.md:3]: Not sure if you want to try the workflow with this release https://github.com/Azure/vscode-kubernetes-tools/pull/854 . If so, you should write the version like `## [1.2.2]` . The `changelog-reader` action only works if the CHANGELOG.md file follows the [Keep a Changelog](https://github.com/olivierlacan/keep-a-changelog) standard for now.

https://keepachangelog.com/en/1.0.0/

**Review by lstocchi — APPROVED** [review]
LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (itowlson) [reply CHANGELOG.md:3]: I've not set up the marketplace tokens and stuff yet so let's do this the old way and switch over to the automated release from the next one (which we can do soon, and can make minimal and use to validate the automated release in the wild).

**Pull request merged**